libraries/botbuilder-dialogs-adaptive-teams/tests/tests/conditionalTests.test.ts

```diff
@@ -0,0 +1,26 @@
+// Licensed under the MIT License.
```

Comment:
Seems that you have nested `tests` folders?

Reply:
Correct. That's to mirror [dotnet](https://github.com/microsoft/botbuilder-dotnet/tree/main/tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Teams.Tests)